cirq-superstaq/cirq_superstaq/ops/qubit_gates.py

```diff
@@ -829,9 +872,17 @@ def _json_dict_(self) -> Dict[str, Any]:
         return cirq.obj_to_dict_helper(self, ["rz_rads"])
 
 
-def custom_resolver(  # pylint: disable=missing-function-docstring
-    cirq_type: str,
-) -> Union[Type[cirq.Gate], None]:
+def custom_resolver(cirq_type: str) -> Union[Type[cirq.Gate], None]:
+    """Tells `cirq.to_json` and `cirq.read_json` how to handle custom cirq-superstaq's custom
+    objects. See https://quantumai.google/cirq/dev/serialization for more information and
+    https://github.com/quantumlib/Cirq/blob/9a1609ce8800ed1aa82bbf6261f5f7b8a41b81cf/cirq-core/cirq/ops/parity_gates_test.py#L309
```

Comment:
would it make sense to just point to `css.ops.qubit_gates_test.test_custom_resolver` and/or `css.serialization`?